### DISCUSSION TITLE
Correspondence of the where clause in schema.json

### DISCUSSION THREAD

**Discussion by mdrm016** [toluaina/pgsync#258] [Q&A]
Is there a way to allow pgsync to filter the data before sending it to Elasticsearch by indicating the filter statement in the scheme.json? 

Example: "Select id, public from my_table where public is true".

**Comment by toluaina** [comment]
I'm afraid not. This question has been asked [here](https://github.com/toluaina/pgsync/issues/251) already.
Its not easy to do this in the config. The best solution right now is to use the plugin

**Reply by toluaina** [reply]
You can perform filter operations using the plugin. This is described [here](https://pgsync.com/tutorial/plugins/)
